db/couchDB.js

```diff
@@ -0,0 +1,64 @@
+const NodeCouchDB = require('node-couchdb');
+const Promise = require('bluebird');
+const couch = require('./couchDBSeed.js');
+
+const addRestaurant = (restaurantID, restaurantName) => {
+	return new Promise((resolve, reject) => {

```

Comment:
Is bluebird required in this file? Does the native javascript Promise constructor work for your purposes here?